src/components/TutorialPage/index.tsx

```diff
@@ -283,17 +283,20 @@ const TutorialPage = (): JSX.Element => {
                     <h1 id="browser-support">Browser support</h1>
                     <ul>
                         <li>
-                            Currently, Simularium supports Firefox, Chrome, and
-                            Edge. Some features may not work on other browsers.
+                            Currently, Simularium supports Firefox, Safari,
+                            Chrome, and Edge. Some features may not work on
```

Comment:
Super nit: Does the ordering of the browsers matter at all here? If this was ordered by popularity, it would be Chrome -> Safari -> Edge -> Firefox. Could do it alphabetically too.